src/tomee_cli/execution.clj

```diff
@@ -19,24 +19,27 @@
   (:require [clojure.java.shell :refer (sh)])
   (:gen-class))
 
+(def tomee-home (System/getenv "TOMEE_HOME"))
```

Comment:
Maybe `tomee-home` should be put in resource.clj or other file. 
We'll use it in other places. So, we only need to have a reference and not declare again.
No make sense I require `execution` in JMS configuration for example to have tomee-home.

Thanks @htmfilho